ci/scripts/driver_weekly.sh

```diff
@@ -73,26 +73,38 @@ git checkout -b "${branch}"
 ######################################################
 # move yaml files from ci/cases/weekly to ci/cases/pr 
 # and push new branch for PR weekly CI tests to GitHub
+REPO_OWNER="emcbot"
+REPO_NAME="global-workflow"
 
 rm -Rf ci/cases/pr
 mv ci/cases/weekly ci/cases/pr
 git add ci/cases
 git commit -m "Moved weekly cases files into pr for high resolution testing"
-git push --set-upstream origin "${branch}"
+
+git remote add upstream "git@github.com:${REPO_OWNER}/${REPO_NAME}.git"
```

Comment:
Recommend using `emcbot` (or maybe just `${REPO_NAME}`) instead of `upstream` for the remote name to avoid confusion since "upsteam" already has a meaning in git parlance.